### ISSUE TITLE
where is pyinn lib to install?

### ISSUE THREAD

**Issue by machanic** [escorciav/roi_pooling#2]
https://github.com/escorciav/roi_pooling/blob/master/roi_pooling/functions/roi_pooling.py#L8
there is `from pyinn.utils import Dtype, Stream, load_kernel`
how to install this lib

**Comment by escorciav** [comment]
Thanks for reaching and your interest in this.

I used conda to install almost everything. For [`pyinn`](https://github.com/szagoruyko/pyinn), I had to rely on `pip`. For reasons that I don't remember, I ended up relying on `pip` to install `Cupy` and `Chainer`, and others.

- The details of the environment are [here](https://github.com/escorciav/roi_pooling/blob/master/environment.yml).

- If you trust `conda`, it's better that you start with the spec-file [here](https://drive.google.com/file/d/1dUfE9aAIPNNvxgavYNQZnF2jRA-0Oc7N/view?usp=sharing). I will push it later (undefined expected time).
This is a good initial point to start installing stuff.
Why? It's well known that `conda` and `pip` are not fully compatible. (At least at the time that I worked on this project).

- Unfortunately, I formatted my machine. Thus, I don't have the environment installed on my computer.

- I may recreate the environment from scratch. Once I do it, I will add an installation section in the README. (undefined expected time)
